students/seongjun/user/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+import json
+
+from django.http    import JsonResponse
+from django.views   import View
+from django.db      import IntegrityError
+
+from user.models        import Account
+from user.validators    import validate_email, validate_password, validate_phone_number
 
 # Create your views here.
+class SignUpView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+
+            if not validate_email(data['email']):
+                return JsonResponse({"message": "이메일 형식을 맞추어주세요"}, status=400)
+            if not validate_password(data['password']):
+                return JsonResponse({'message': 'Password 8이상 작성해야합니다.'}, status=400)
+            if validate_phone_number(data['phone_number']) == False:
+                return JsonResponse({'message': '잘못된 형식의 번호입니다.'}, status=400)
+            data['phone_number'] = validate_phone_number(data['phone_number'])
```

Comment:
즉, 위의 피드백을 반영한다면, 바로 phone_number 의 를 value 로 받아서 다시 data['phone_number'] 에 할당하는 것 대신

```suggestion
            if validate_phone_number(data['phone_number']):
                return ~~~~
```

위와 같이 작성해주시는 것이 더 가독성이 좋아보입니다.

Reply:
확인햇습니다!